core/src/main/resources/reference.conf

```diff
@@ -158,8 +158,7 @@ akka.persistence.r2dbc {
     # Configures the statement cache size.
     # 0 means no cache, negative values will select an unbounded cache
     # a positive value will configure a bounded cache with the passed size.
-    # TODO: add more elaborated description explaining when and why to use a cache
-    statement-cache-size = 100
+    statement-cache-size = 5000
```

Comment:
Increasing this because not using ? param any more for the slice so there could be more separate statements. We don't really want to evict any statements from the cache but I think this was introduced because of some issue with the unbounded cache.